static/src/js/components/FooterLink/FooterLink.scss

```diff
@@ -23,30 +23,30 @@
   }
 
   &__info-bit--emph {
-    color: $color__white;
-    text-decoration-color: $color__white;
+    color: $color__spacesuit-white;
+    text-decoration-color: $color__spacesuit-white;
   }
 
   &__info-link {
     white-space:nowrap;
-    color: $color__blue--light;
+    color: $color__spacesuit-white;
 
     .footer-link__info-bit--emph & {
-      color: $color__white;
-      text-decoration-color: $color__white;
+      color: $color__spacesuit-white;
+      text-decoration-color: $color__spacesuit-white;
 
       &:hover {
-        color: $color__white;
-        text-decoration-color: $color__white;
+        color: $color__spacesuit-white;
+        text-decoration-color: $color__spacesuit-white;
       }
     }
 
     &:hover {
-      color: $color__blue--light;
+      color: $color__spacesuit-white;
```

Comment:
is this hover the same color as the non hover?

Reply:
Yes, the footer link text is white.